packages/care-ops-auth/providers/workos.js

```diff
@@ -35,29 +35,24 @@ export class WorkosAuthProvider extends AuthProvider {
     return clientId;
   }
 
-  async _initClient(clientId) {
-    return new Promise(resolve => {
-      const clientConfig = {
-        redirectUri: location.origin + AuthProvider.PATH_AUTHD,
-        onRedirectCallback: ({ user, state }) => {
-          const path = state;
-          if (!user) {
-            this.loginPrompt(path);
-            return;
-          }
-
-          this.handleAuthedPath(path);
-
-          resolve();
-        },
-        ...this.config.createClientOptions,
-      };
-
-      createClient(clientId, clientConfig)
-        .then(client => {
-          this.client = client;
-        });
-    });
+  async _initClient(clientId, success) {
+    const clientConfig = {
+      redirectUri: location.origin + AuthProvider.PATH_AUTHD,
+      onRedirectCallback: ({ user, state }) => {
+        const path = state;
+        if (!user) {
+          this.loginPrompt(path);
+          return;
+        }
+
+        this.handleAuthedPath(path);
+
+        success();
```

Comment:
the only change here is unwrapping the `new Promise` and changing the resolve to success